Description: Use cups-filter's filters, type declarations and conversions.
 This fixes the tests that need those.
Origin: vendor
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2012-10-13
--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -373,6 +373,30 @@
 		ln -s /usr/share/cups/data/psglyphs $root/data
 	fi
 	ln -s /usr/share/cups/fonts /tmp/cups-$user/share
+
+	#
+	# cups-filters 1.0.24
+	#
+	ln -s /usr/share/cups/mime/cupsfilters.types /tmp/cups-$user/share/mime
+	ln -s /usr/share/cups/mime/cupsfilters.convs /tmp/cups-$user/share/mime
+
+	ln -s /usr/lib/cups/filter/bannertopdf /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/commandtoescpx /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/commandtopclx /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/imagetopdf /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/imagetops /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/imagetoraster /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/pdftoijs /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/pdftoopvp /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/pdftopdf /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/pdftops /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/pdftoraster /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/pstopdf /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/rastertoescpx /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/rastertopclx /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/textonly /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/texttopdf /tmp/cups-$user/bin/filter
+	ln -s /usr/lib/cups/filter/texttops /tmp/cups-$user/bin/filter
 fi
 
 #
